src/modules/server/server.config.ts

```diff
@@ -1,8 +1,7 @@
 import { Transform } from 'class-transformer';
 import { IsArray, IsBoolean, IsNumber, IsString, IsUrl } from 'class-validator';
-import { RedisConfig } from '../../infra/redis/redis.config.js';
 
-export class ServerConfig extends RedisConfig {
```

Comment:
Ich schätze das die "RedisConfig" als Interface hier wieder rein wandern wird. Wenn man den "Redis Service" in factories zerlegt.